core-web/libs/portlets/edit-ema/portlet/src/lib/edit-ema-editor/components/edit-ema-toolbar/edit-ema-toolbar.component.html

```diff
@@ -80,7 +83,11 @@
         </div>
     </p-toolbar>
     <dot-ema-info-display
-        *ngIf="!es.editorData.canEditPage || es.editorData.mode !== editorMode.EDIT"
+        *ngIf="
+            !es.editorData.canEditPage ||
+            (es.editorData.mode !== editorMode.EDIT &&
+                es.editorData.mode !== editorMode.INLINE_EDITING)
+        "
```

Comment:
I added one more field to the editorData for this
It works, but requires refactoring of the tests and there may be some side effects
(editorData is used in several places)

I'm not sure if we should include that in this ticket, thoughts? @fmontes @zJaaal

Reply:
A selector don't required changes in the state object

Reply:
Oh, I thought we were talking about editing the editorState$ selector.
This would involve changing tests and 

But we can add a new selector for the toolbar, starting from the editorState

Like:

```
    readonly editorToolbarData$ = this.select(this.editorState$, (editorState) => ({
        ...editorState,
        showWorkflowActions:
            editorState.editorData.mode === EDITOR_MODE.EDIT ||
            editorState.editorData.mode === EDITOR_MODE .INLINE_EDITING,
        showInfoDisplay:
            !editorState.editorData.canEditPage ||
            (editorState.editorData.mode !== EDITOR_MODE.EDIT &&
                editorState.editorData.mode !== EDITOR_MODE.INLINE_EDITING)
    }));
```
I will add this